pkg/adapter/containers.go

```diff
@@ -694,3 +694,45 @@ func (r *LocalRuntime) UnpauseContainers(ctx context.Context, cli *cliconfig.Unp
 	}
 	return pool.Run()
 }
+
+// Cleanup any leftovers bits of stopped containers
+func (r *LocalRuntime) CleanupContainers(ctx context.Context, cli *cliconfig.CleanupValues) ([]string, map[string]error, error) {
+	var (
+		ok       = []string{}
+		failures = map[string]error{}
+	)
+
+	ctrs, err := shortcuts.GetContainersByContext(cli.All, cli.Latest, cli.InputArgs, r.Runtime)
```

Comment:
I still don't know why we actually support --all and --latest for cleanup... but since it made it into the interface, we can't really remove it.

Reply:
You can just remove the documentation and hide these fields.  I don't believe that anyone actually ever runs container cleanup directly. We could probably hide the entire implementation from users.